catalog/models.py

```diff
@@ -106,9 +104,6 @@ def bind_fields(self):
             .prefetch_related('page__images')
         )
 
-    def get_offset(self, start, step):
```

Comment:
This method seems pretty convenient. Maybe not the best name, but we can simply rename it: `def slice(self, offset, limit) -> QuerySet`

Reply:
@artemiy312 , queryset already inherited from python standard slice mech: iterable[a:b:c].
Here we throw away this standard mech (in one specific place, by the way) just to cut `offset +` symbols. So, let's rm it in favour of standard mech?